### PR TITLE
Example dag twitter

### PR DESCRIPTION
**Pull request by ekhtiar**
Recommitting my Example Dag on twitter with corrections requested from previous PR.

### CODE REVIEW

File: airflow/example_dags/example_twitter_dag.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+yesterday = date.today() - timedelta(days=1)
+dt = yesterday.strftime("%Y-%m-%d")
+
+for channel in to_channels:
+
+    file_name = "to_" + channel + "_" + yesterday.strftime("%Y-%m-%d") + ".csv"
+
+    load_to_hdfs = BashOperator(
+        task_id="put_" + channel + "_to_hdfs",
+        bash_command="HADOOP_USER_NAME=hdfs hadoop fs -put -f /root/tweets/" + file_name + " /user/admin/tweets/" + channel + "/",
```
Comment (bolkedebruin) [inline airflow/example_dags/example_twitter_dag.py]: why hardcode these user locations? Maybe use a configurable location?

File: airflow/example_dags/example_twitter_dag.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+# The purpose of this is to give you a sample of a real world example DAG!
+# ----------------------------------------------------------------------------------------------------------------------
+
+# ----------------------------------------------------------------------------------------------------------------------
+# Load The Dependencies
+# ----------------------------------------------------------------------------------------------------------------------
+
+import sys
+
+reload(sys)
```
Comment (bolkedebruin) [inline airflow/example_dags/example_twitter_dag.py]: Why are you doing this?

**Comment by bolkedebruin** [comment]
Ah sorry for the comments on the outdated diffs. Can you maybe see if you can change the location to a configurable location or a location that is writeable by default?

**Comment by ekhtiar** [comment]
Not a problem. Yep I will make it more configurable / dynamic. Coming right up :)

**Comment by jlowin** [comment]
@ekhtiar the unit tests are failing because there is one that checks the number of example dags. Please bump up the number near the top of `airflow/tests/core.py`. In addition, all example dags are now unit tested and it looks like this one takes a very long time to run and not all tasks are succeeding. I'm sure that's the hive/hdfs bits. It's a very useful practical example though, so I think we should remove it from the unit tests -- please add it to the `skip_tests` list here: https://github.com/airbnb/airflow/blob/master/tests/jobs.py#L49

**Comment by ekhtiar** [comment]
Hi, I made the code more dynamic. Also the default directories are changed to tmp files, to which user should have permission to by default. I also bumped up the example dags number, and added this to the skip test list. I am pretty confused about why tests are failing, can someone please help.

**Comment by bolkedebruin** [comment]
There is no issue. Coveralls is failing again...

**Comment by bolkedebruin** [comment]
@ekhtiar the code seems ok now. Thanks, great work! Now squash your commits into 1 single commit and use the following guidelines for your commit message: http://chris.beams.io/posts/git-commit/ . This means better subject, short description on what and why (not how). Suggestion for the subject/title of your commit is "Add twitter feed example dag". 

This is required to make the history readable when we create a changelog and release notes.

**Comment by ekhtiar** [comment]
I read it up and tried to follow that and your guideline. I think I did it right.

**Comment by bolkedebruin** [comment]
👍

**Pull request merged**